src/background/index.js

```diff
@@ -132,8 +132,11 @@ chrome.tabs.onRemoved.addListener((tabId) => {
   tabStats.delete(tabId);
 });
 
+// Initialize the value of the options upfront
+store.get(Options);
```

Comment:
is this sync?

Reply:
Yes, [more info here](https://hybrids.js.org/#/store/usage?id=get) ;)

Reply:
I've meant if the side effect (loading of data) is sync, but now I see it is not. To be sure data is loaded we should use `await store.resolve(store.get(Options))`, right?
My question is if we wont fall into some race condition.

Reply:
In the matter of possible race conditions, the current code works exactly the same as the previous one (before the refactor - look here https://github.com/ghostery/ghostery-dnr-extension/commit/72dca21dcc7684e6006979b732da73ed659b7166#diff-f2f25f467270cf52f2aaf891a1f437073f8ef727859a636251285ff26d886f4eL78). 

The options are get (before they were updated), which triggers fetching them. I wanted to avoid logic change as if we want to wait, then we have to change the value, which the listener returns for half of the code inside.